src/test/java/org/sagebionetworks/web/unitclient/widget/entity/tabs/DatasetsTabTest.java

```diff
@@ -253,18 +252,6 @@ public void testSetTabClickedCallback() {
     verify(mockTab).addTabClickedCallback(mockOnClickCallback);
   }
 
-  @Test
-  public void testClickTable() {
-    // verify that clicking on an item in the datasets list sends the event back to the entity page top
-    // (to get the new target entity)
```

Comment:
We were testing code that was never executed during runtime (because we use a standard click handler on the table list item links!)